keyboards/paulpcb/readme.md

```diff
@@ -0,0 +1,27 @@
+# paulpcb
+
+![paulpcb](https://i.imgur.com/BZneAHS.jpeg)
+![AWL Paul keyboard](https://i.imgur.com/dRibVfa.jpeg)
```

Comment:
```suggestion
![paulpcb](https://i.imgur.com/BZneAHSh.jpeg)
![AWL Paul keyboard](https://i.imgur.com/dRibVfah.jpeg)
```

Reply:
I can see that this changes the resolution of the image, is this standard practice or written anywhere or personal preference? I couldn't see it in the PR Checklists.

Reply:
In general you should avoid making people download massive images.
![image](https://user-images.githubusercontent.com/4781841/132916824-e6b69963-4cee-4ef2-8038-263407cce688.png)